integrationTests/vm/arwen/utils.go

```diff
@@ -137,15 +149,63 @@ func (context *TestContext) DeploySC(wasmPath string, parametersString string) {
 
 	err := context.TxProcessor.ProcessTransaction(tx)
 	if err != nil {
-		require.FailNow(context.T, err.Error())
+		return err
+	}
+
+	err = context.GetSilentSCProcessorError()
+	if err != nil {
+		return err
+	}
+
+	owner.Nonce++
+
```

Comment:
i think here there is a need to save the account. owner.

Reply:
Actually, the saving of the account is performed under the hood at processing time. The nonce increment there is for the `testParticipant` structure - so that we do not loose track of the nonce in the test scenario. OK?